anagram/anagram.py

```diff
@@ -1,2 +1,12 @@
 def find_anagrams(word, candidates):
-    pass
+    is_anagram = []
+    for c in candidates:
+        if len(c) == len(word):
```

Comment:
As I understand it, if the lengths are not equal then the function just returns an empty list, is that correct? Did you consider adding an else statement to maybe throw an error or something if that wasn't the case?

Reply:
My reasoning is that if the lengths are not equal, the candidate doesn't enter the stage in which the lists are created, filled and compared, so there is no way it can enter the list of anagrams. Does it seem correct?